NEWS.md

```diff
@@ -10,6 +10,7 @@
 * [MODSOURCE-341](https://issues.folio.org/browse/MODSOURCE-341) Store MARC Holdings record
 * [MODSOURCE-332](https://issues.folio.org/browse/MODSOURCE-332) Improve performance of left-anchored SRS queries
 * [MODSOURCE-343](https://issues.folio.org/browse/MODSOURCE-343) Support MARC Holdings in API
+* [MODSOURCE-261](https://issues.folio.org/browse/MODSOURCE-261) Cover kafka handlers with tests and fix ignored
```

Comment:
please move this entry to the new development iteration v5.3.0-SNAPSHOT, v5.2.0 has been released already